lib/module.js

```diff
@@ -59,6 +59,7 @@ function axiosModule (_moduleOptions) {
     progress: true,
     proxyHeaders: true,
     proxyHeadersIgnore: ['accept', 'host', 'cf-ray', 'cf-connecting-ip', 'content-length', 'content-md5', 'content-type'],
+    proxyCookies: true,
```

Comment:
I would suggest turning of by default because of security implications it has (private SSR API calls may leak headers)